### ISSUE TITLE
No longer packaged data-files cause test failures

### ISSUE THREAD

**Issue by loqs** [mopidy/mopidy#2173]
**Describe the bug**
Since https://github.com/mopidy/mopidy/commit/80e318805fc9a68c3cf6c6e2d468612ba7ab24e6 data-files (such as those previously included by `mopidy/http/data *`) are no longer included in the built package which causes test failures if tests are conducted against the built package.

**How to reproduce**
```
$ SETUPTOOLS_SCM_PRETEND_VERSION=4.0.0a1 python -m build --wheel --no-isolation
$ python -m venv --system-site-packages test-env
$ test-env/bin/python -m installer dist/*.whl
$ test-env/bin/python -m unittest discover -vs .
....
======================================================================
FAIL: test_should_redirect_to_clients_list (tests.http.test_server.HttpServerWithInvalidDefaultApp.test_should_redirect_to_clients_list)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python3.13/site-packages/tornado/testing.py", line 268, in _callTestMethod
    result = method()
  File "/build/mopidy/src/mopidy-4.0.0a1/tests/http/test_server.py", line 397, in test_should_redirect_to_clients_list
    assert "This web server is a part of the Mopidy music server." in body
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError

======================================================================
FAIL: test_should_return_index (tests.http.test_server.MopidyAppTest.test_should_return_index)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python3.13/site-packages/tornado/testing.py", line 268, in _callTestMethod
    result = method()
  File "/build/mopidy/src/mopidy-4.0.0a1/tests/http/test_server.py", line 68, in test_should_return_index
    assert "This web server is a part of the Mopidy music server." in body
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError

======================================================================
FAIL: test_should_return_static_files (tests.http.test_server.MopidyAppTest.test_should_return_static_files)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python3.13/site-packages/tornado/testing.py", line 268, in _callTestMethod
    result = method()
  File "/build/mopidy/src/mopidy-4.0.0a1/tests/http/test_server.py", line 83, in test_should_return_static_files
    assert "html {" in response.body.decode()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError

----------------------------------------------------------------------
Ran 920 tests in 1.105s

FAILED (failures=3)

```
Full build and check logs in additional context section.
**Expected behaviour**
All tests for the installed package should pass.

**Environment**
Please complete the following information:
- Operating system (e.g. Debian Buster, Ubuntu 20.04)
Arch Linux - Rolling Release
python 3.13.0
- [Running Mopidy](https://docs.mopidy.com/en/latest/running/) as a service or in the terminal?
Not Applicable.
- Your config (output of `sudo mopidyctl config`)
Not Applicable. 
- Software versions (output of `sudo mopidyctl deps`)
Not Applicable. 
- Any specific hardware (e.g. an exotic USB soundcard)
Not Applicable.

**Additional context**
- https://gitlab.archlinux.org/archlinux/packaging/packages/mopidy/-/issues/1 covers mopidy 3.4.2 failing to build with python 3.13 which was fixed in https://github.com/mopidy/mopidy/commit/136c6f435eafecb56d6b56d970db12c098cd88a2 but that commit is not trivial to back-port.
- https://gitlab.archlinux.org/archlinux/packaging/packages/mopidy/-/merge_requests/2
- [mopidy-4.0.0a1-1-x86_64-build.log](https://github.com/user-attachments/files/18011377/mopidy-4.0.0a1-1-x86_64-build.log)
- [mopidy-4.0.0a1-1-x86_64-check.log](https://github.com/user-attachments/files/18011381/mopidy-4.0.0a1-1-x86_64-check.log)

**Comment by jodal** [comment]
Thanks for the report! I suspected something like this was wrong, and I really needed the kick to finally sit down and look at this.

After reading up on setuptools support for data files at https://setuptools.pypa.io/en/latest/userguide/datafiles.html and trying another approach first, I ended up using exactly your patch. Looking at the old `MANIFEST.in` file in removed in 80e318805fc9a68c3cf6c6e2d468612ba7ab24e6 this looks to be exactly the files needed at runtime.

As confirmation that this solved the problem, our [`ubuntu-devel`](https://github.com/mopidy/mopidy/actions/workflows/ubuntu-devel.yml) CI workflow is now green, for the first time since the above mentioned commit. :see_no_evil:
